rollup.config.js

```diff
@@ -47,37 +86,30 @@ const plugins = [
   commonjs({
     ignoreGlobal: true,
   }),
-  !esbundle && replace({
-    'process.env.NODE_ENV': JSON.stringify(prod ? 'production' : 'development'),
-  }),
-  prod && inject({
-    process: processShim,
-  }),
-  babel({
-    babelrc: false,
-    presets: [
-      ['env', { modules: false, loose: true }],
-      'react',
-    ],
-    plugins: [
-      !prod && 'flow-react-proptypes',
-      prod && 'transform-react-remove-prop-types',
-      'transform-flow-strip-types',
-      'external-helpers',
-      'transform-object-rest-spread',
-      'transform-class-properties',
-    ].filter(Boolean),
-  }),
+  UMD &&
+    replace({
+      'process.env.NODE_ENV': JSON.stringify(
+        PRODUCTION ? 'production' : 'development',
+      ),
+    }),
+  PRODUCTION &&
+    inject({
+      process: processShim,
+    }),
+  babel(),
 ].filter(Boolean)
 
-if (prod) plugins.push(uglify(), visualizer({ filename: './bundle-stats.html' }))
+if (PRODUCTION) {
+  plugins.push(uglify(), visualizer({ filename: './bundle-stats.html' }))
+}
 
 export default {
```

Comment:
Just a quick note, you can use an array here to avoid bloating the package scripts 😬

Not sure if you explicitly decided against it

Reply:
just went with the approach already used in the repo, can refactor to array later